docs/1-introduction/1-1-getting-started/1-1-3-hello-triangle.md

```diff
@@ -431,11 +431,11 @@ We begin by filling a `bufferInfo` descriptor for our buffer, we specify how man
 
 And finally we create a `resourceData`, and populate the only field we care about: `pSysMem`, which is a pointer to our vertices which are currently in system RAM.
 
-Then we issue the creation of the buffer using `CreateBuffer`, using the informations we collected until now.
+Then we issue the creation of the buffer using `CreateBuffer`, using the information we collected until now.
 
 ## Drawing our triangle
 
-Finally we have reached the moment of truth, in our `Render` function we will add a few things:
+Finally, we have reached the moment of truth, in our `Render` function we will add a few things:
```

Comment:
`Finally` comes right after this one again, maybe turn `Finally` here into a `Now`